flatpak-node-generator.py

```diff
@@ -0,0 +1,1675 @@
+#!/usr/bin/env python3
```

Comment:
Please don't bundle this script. If there are app-specific changes to it - submit them upstream instead.